submissions/benjaminKamena/phpDataase/ordersQueryDb.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+include('connected.php');
+ if( isset($_GET['id']) && strlen($_GET['id'])> 0){
+ $id =  $_GET['id'];
+}
+
+try{
+	$sql = "SELECT order_items.quantity, order_items.unit_price, 
+            order_items.total_amount, orders.order_date, orders.id,customers.street_address, customers.city,customers.state, 
+            customers.country,products.name 
+            FROM order_items 
+            LEFT JOIN orders ON orders.id = order_items.order_id
+            LEFT JOIN customers on orders.customer_id = customers.id
+            LEFT JOIN products ON products.id = order_items.product_id 
+            WHERE orders.customer_id = '$id' ";
+            $stmt = $pdo->query($sql);
+            $orders = $stmt->fetchAll();
```

Comment:
Kindly separate your query from your view. Just the way you did for this previous task,